platform/security/sts/security-sts-ldapclaimshandler/src/main/java/ddf/security/sts/claimsHandler/LdapClaimsHandler.java

```diff
@@ -219,4 +219,25 @@ public void setKerberosRealm(String kerberosRealm) {
   public void setKdcAddress(String kdcAddress) {
     this.kdcAddress = kdcAddress;
   }
+
+  @VisibleForTesting
+  BindRequest selectBindMethod() {
+    return BindMethodChooser.selectBindMethod(
+        bindMethod, bindUserDN, bindUserCredentials, kerberosRealm, kdcAddress);
+  }
+
+  @VisibleForTesting
```

Comment:
✏️ Wish we could do something to avoid having so many new `@VisibleForTesting` methods. Should we consider extracting a class that delegates to `AttributeMapLoader` and inject that instead?